### DISCUSSION TITLE
encountering  problem while using "make -j6 command"

### DISCUSSION THREAD

**Discussion by saqibhayat556** [arfc/moltres#240] [General / Ideas]
I have installed Moose into my Ubuntu environment. I have all the test and they completed successfully now the problem i am facing is that i have tried to run the example given in MOOSE presentation.
'cd ~/projects/moose/tutorials/darcy-thermo_mech/step01_diffusion
make -j 6 # use number of processors for your system'
i get this error
![Screenshot 2023-07-08 10:50:46](https://github.com/arfc/moltres/assets/124503670/10e0ba37-6492-42a9-9a22-2d52b3490023)

**Comment by smpark7** [comment]
Hi @saqibhayat556
What exact steps did you take to "install Moose"? I can't replicate your error. I'll try to help you, but I also recommend posting this error on the [MOOSE discussions page](https://github.com/idaholab/moose/discussions) because it involves MOOSE's tutorial.

**Reply by js-jixu** [reply]
I encountered a similar problem today. I installed Molteres on an Ubuntu 20.04 server today according to the tutorial on the [Molteres homepage](https://smpark7.github.io/moltres/getting_started/installation.html). After successfully downloading all the files, I compiled it using the "make -j8" command. The following errors occurred during the compilation process:

![error](https://github.com/arfc/moltres/assets/99813361/8b4ced61-a151-4b6a-950e-3c2dd5f1ba95)

The following are the historical commands from my entire installation process, and I believe I did not make any mistakes:

```
    1  curl -L -O https://github.com/conda-forge/miniforge/releases/latest/download/Mambaforge-Linux-x86_64.sh
    2  bash Mambaforge-Linux-x86_64.sh -b -p ~/mambaforge3
    3  export PATH=$HOME/mambaforge3/bin:$PATH
    4  mamba init
    5  exit
    6  conda config --add channels https://conda.software.inl.gov/public
    7  mamba install moose-tools moose-libmesh
    8  mkdir ~/projects
    9  cd ~/projects
   10  git clone https://github.com/arfc/moltres.git
   11  cd moltres
   12  git checkout master
   13  git submodule init
   14  git submodule update
   15  conda activate moose
   16  mamba activate moose
   17  mamba create -n moose moose-dev
   18  mamba activate moose
   19  mamba install moose-tools moose-libmesh
   20  make -j8
   21  history
```

I think it may be that MOOSE has made some updates that are not compatible with Moltres. When I separately compile the `/framework` folder in the MOOSE submodule in Moltres, this error also occurs. In addition, I downloaded and compiled MOOSE (excluding the Moltres section) according to the tutorials on the MOOSE official website, and the compilation and testing process of MOOSE was very smooth at this time.

The error log is attached.

[error_log.txt](https://github.com/arfc/moltres/files/12369451/error_log.txt)

**Reply by js-jixu** [reply]
My problem has been solved by this [discussion](https://github.com/idaholab/moose/discussions/25240).😁

**Reply by smpark7** [reply]
Hey @js-jixu, just to note, if you follow their steps, your installation will be with the latest version of MOOSE. If you want the old MOOSE version from March, you'll need to find the corresponding version of moose-tools and moose-libmesh [here](https://conda.software.inl.gov/public/linux-64/).

**Reply by js-jixu** [reply]
Thank you for your reminding, I'll reinstall moose-tools.
